pkg/templates/templates_generated.go

```diff
@@ -5399,6 +5399,16 @@ write_files:
     #EOF
 {{- end}}
 
+{{- if ShouldConfigureCustomCATrust}}
+{{range $i, $cert := GetCustomCATrustConfigCerts}}
+- path: /usr/local/share/ca-certificates/00000000000000cert{{$i}}.crt
```

Comment:
To match the logic of binary used by pod that removes/updates certs, a prefix of length 14 is required. We want these initial certs to be removed when update comes, so for now i just added all zeros. 
Is there some more elegant way of doing that maybe?